src/Tonic/AudioFileUtils.cpp

```diff
@@ -91,7 +91,7 @@ namespace Tonic {
   
   SampleTable loadAudioFile(string path, int numChannels){
     Tonic::error("loadAudioFile is currently only implemented for Apple platforms.");
-    return NULL;
+    return SampleTable();
```

Comment:
NULL is defined ((void*) 0) in Windows. This is of course a compiler error. If we return an empty buffer instead, we also avoid the crashing of the app which calls loadAudioFile().